src/searchengine/searchengine.cpp

```diff
@@ -409,6 +414,8 @@ void SearchEngine::updateNova() {
     }
 }
 
+#define SET_SEARCH_STATUS(text) \
+    activeSearchTab->search_status = tr(text);
```

Comment:
This macro is totally useless. Just use `activeSearchTab->search_status = tr(text);` directly.